cypress/support/helpers/runTestsForPage.js

```diff
@@ -56,8 +57,6 @@ const runTestsForPage = ({
             variant: config[service].variant,
           };
 
-          const ampOnlyServices = ['news', 'sport', 'newsround'];
-
```

Comment:
Nice one!
Just one query: `ampOnlyServices` is being imported on L21, does it need to be re-declared here? Or am I missing something?

Reply:
That line has been deleted @HarryVerhoef, I think it's fine to pull it in from the global scope on line 61?

Reply:
Doesn't seem to need to be redeclared.

Reply:
My bad haha, may have interpreted those "-"s as "+"s 😂 Thanks for the quick response